src/screens/PutawayList/index.tsx

```diff
@@ -141,12 +139,15 @@ class PutawayList extends React.Component<Props, State> {
     });
   };
   onPutAwayTapped = (putAway: PutAway) => {
-    this.props.navigation.navigate('PutawayDetails', {
-      putAway: putAway,
-      exit: () => {
-        this.props.navigation.navigate('PutawayList');
-      }
-    });
+    console.log('PUT AWAY :::', JSON.stringify(putAway));
+    if (putAway?.putawayItems[0]?.quantity > 1) {
```

Comment:
Why this check `putAway?.putawayItems[0]?.quantity > 1`? I don't understand the idea behind it.